llvm/lib/CodeGen/ValueTypes.cpp

```diff
@@ -237,8 +237,22 @@ Type *EVT::getTypeForEVT(LLVMContext &Context) const {
     return FixedVectorType::get(Type::getInt1Ty(Context), 3);
   case MVT::v4i1:
     return FixedVectorType::get(Type::getInt1Ty(Context), 4);
+  case MVT::v5i1:
+    return FixedVectorType::get(Type::getInt1Ty(Context), 5);
+  case MVT::v6i1:
+    return FixedVectorType::get(Type::getInt1Ty(Context), 6);
+  case MVT::v7i1:
+    return FixedVectorType::get(Type::getInt1Ty(Context), 7);
```

Comment:
Can we not update this method to use GenVT.inc for most cases?

Reply:
Let me try to reduce that, I guess most of them can defer until the RISC-V patch

Reply:
I guess I may misunderstand you comment yesterday, anyway, I created another PR for that https://github.com/llvm/llvm-project/pull/96608